snorkel/utils.py

```diff
@@ -145,3 +146,124 @@ def tokens_to_ngrams(tokens, n_max=3, delim=' '):
     for root in range(N):
         for n in range(min(n_max, N - root)):
             yield delim.join(tokens[root:root+n+1])
+
+class SnorkelPrint:
```

Comment:
Can we rename this class to `CandidateViewer` or something more descriptive?

Reply:
Agreed, I changed the name to `CandidateViewer`. Feel free to change the name to a better or more descriptive one.